scripts/crowdin/pretranslate.mjs

```diff
@@ -0,0 +1,159 @@
+const projectId = process.env.CROWDIN_PROJECT_ID;
+const token =  process.env.CROWDIN_PERSONAL_TOKEN;
+
+const BASE_URL = `https://api.crowdin.com/api/v2/projects/${projectId}`;
+const preTranslateURL = `${BASE_URL}/pre-translations`;
+const REQUEST_INTERVAL_TIMEOUT = 10_000;
+const MAXIMUM_REQUEST_RETRIES = 12;
+
+const requestData = {
+  method: 'mt',
+  engineId: 410374,
+  autoApproveOption: 'all',
+  duplicateTranslations: false,
+  skipApprovedTranslations: true,
+  translateUntranslatedOnly: true,
+  translateWithPerfectMatchOnly: false,
+};
+
+const getMachineTranslationEngineID = async () =>{
+  try {
+    const response = await fetch('https://api.crowdin.com/api/v2/mts', {
```

Comment:
Why some of api endpoinst are defined as const but here you've used hardcoded value directly?

Reply:
Fixed